staging/src/k8s.io/client-go/tools/record/event.go

```diff
@@ -263,6 +263,8 @@ func recordEvent(sink EventSink, event *v1.Event, patch []byte, updateExistingEv
 	case *errors.StatusError:
 		if errors.IsAlreadyExists(err) {
 			klog.V(5).Infof("Server rejected event '%#v': '%v' (will not retry!)", event, err)
+		} else if errors.HasStatusCause(err, v1.NamespaceTerminatingCause) {
```

Comment:
If the log lines are the same in both cases, why not change line 264 to be:

```golang
if errors.IsAlreadyExists(err) || errors.HasStatusCause(err, v1.NamespaceTerminatingCause) {
```

Reply:
updated to combine both conditions